### DISCUSSION TITLE
Nautilus theme change

### DISCUSSION THREAD

**Discussion by paradisedj** [swayos/swayos.github.io#16] [Q&A]
I love your swayos.
Is there a way to change the nautilus file manager themes?
I have tried with Gnome-Tweaks and lxapperance and none of theme worked.

**Comment by milgra** [comment]
Hi, it became a GTK4 app, try gradience or install a different theme as described here : https://www.reddit.com/r/gnome/comments/yvglat/nautilus_theme_cant_be_changed_in_gnome_43/?utm_source=share&utm_medium=ios_app&utm_name=iossmf
